s.go

```diff
@@ -11,7 +11,10 @@ import (
 	_ "github.com/zquestz/s/providers/dockerhub"
 	_ "github.com/zquestz/s/providers/duckduckgo"
 	_ "github.com/zquestz/s/providers/github"
+	_ "github.com/zquestz/s/providers/go"
 	_ "github.com/zquestz/s/providers/google"
+	_ "github.com/zquestz/s/providers/npm"
+	_ "github.com/zquestz/s/providers/npmsearch"
```

Comment:
Appears you didn't load the youtube plugin here. Once that is added I will merge.